zerver/views/custom_profile_fields.py

```diff
@@ -78,6 +78,14 @@ def is_default_external_field(field_type: int, field_data: ProfileFieldData) ->
     return True
 
 
+def updating_subtype_of_external_account(
+    existing_field_data: ProfileFieldData, requested_field_data: ProfileFieldData
+) -> bool:
+    if existing_field_data["subtype"] != requested_field_data["subtype"]:
+        return True
+    return False
```

Comment:
You can write `return existing_field_data["subtype"] != requested_field_data["subtype"]`